oracle/src/events/processTransfers/index.js

```diff
@@ -37,6 +41,20 @@ function processTransfersBuilder(config) {
 
         logger.info({ from, value }, `Processing transfer ${transfer.transactionHash}`)
 
+        const receipt = await web3Foreign.eth.getTransactionReceipt(transfer.transactionHash)
+        const existsAffirmationEvent = receipt.logs.some(
+          e => e.address === config.foreignBridgeAddress && e.topics[0] === userRequestForAffirmationHash
+        )
+
+        if (existsAffirmationEvent) {
+          logger.info(
+            `Transfer event discarded because UserRequestForAffirmation event is also present in the same transaction ${
```

Comment:
do you think the message like "Transfer event discarded because a transaction with alternative receiver detected" is more informative?

Reply:
Yes, thanks, updated https://github.com/poanetwork/tokenbridge/pull/221/commits/df18bf6c409d15e36e7e1864b524b0c9488dd817